bundles/org.openhab.io.homekit/README.md

```diff
@@ -676,7 +676,7 @@ Support for this is planned for the future release of openHAB HomeKit binding.
 |                      |                             | Brightness                   | Dimmer, Color                 | Brightness in % (1-100). See "Usage of dimmer modes" for configuration details.                                                                                                                                                                                                                                                                     |
 |                      |                             | ColorTemperature             | Number                        | Color temperature represented in reciprocal megaKelvin. The default value range is from 50 to 400. Color temperature should not be used in combination with hue, saturation and brightness. It supports following configuration parameters: minValue, maxValue                                                                                      |
 | Fan                  |                             |                              |                               | Fan                                                                                                                                                                                                                                                                                                                                                 |
-|                      | ActiveStatus                |                              | Switch                        | Accessory current working status. A value of "ON"/"OPEN" indicates that the accessory is active and is functioning without any errors.                                                                                                                                                                                                              |
+|                      | ActiveStatus                |                              | Switch, Dimmer                | Accessory current working status. A value of "ON"/"OPEN" indicates that the accessory is active and is functioning without any errors.                                                                                                                                                                                                              |
```

Comment:
please add Dimmer as supported item to ActiveStatus of other characteristics. if they use the same booleanitemreader. 
at least for HeaterCooler and Valve i could think of an use cases.